### DISCUSSION TITLE
Date Slider? Slider Labels?

### DISCUSSION THREAD

**Discussion by bryanwhiting** [h2oai/wave#1788] [Ideas]
The date picker is good, but it's slow from a UI perspective (lots of clicking) so I've resorted to using two dropdowns for min/max dates. It'd be great to have a date slider or a way to label the slider values:

```
ui.slider(values = [1, 2, 3], labels = ["2022-01-01", "2022-02-02", "2022-01-03"])
```

**Comment by vopani** [comment]
https://github.com/h2oai/wave/issues/1012
